Formula/d/dotnet.rb

```diff
@@ -92,13 +114,10 @@ def install
       buildpath.install resource("release.json")
     end
 
+    args << "-p:PublishReadyToRun=false" if OS.linux? && Hardware::CPU.intel?
+
```

Comment:
```suggestion
```

Try removing this after 
- https://github.com/Homebrew/homebrew-core/pull/260700